### PR TITLE
feat(apple): Add custom performance metrics

### PR DESCRIPTION
**Pull request by philipphofmann**
Only merge after releasing Cocoa 7.28.0. Related Cocoa PR https://github.com/getsentry/sentry-cocoa/pull/2268.
This PR is similar to https://github.com/getsentry/sentry-docs/pull/5604.

### CODE REVIEW

**Review by markushi — APPROVED** [review]
LGTM 👍

File: src/platform-includes/performance/automatic-performance-metrics/apple.mdx [hunk 1]
```diff
@@ -0,0 +1,4 @@
+If configured, the Cocoa SDK automatically collects the following performance metrics:
+
+* Cold and warm app start time - learn more in <PlatformLink to="/performance/instrumentation/automatic-instrumentation/#app-start-tracking">App Start Tracking</PlatformLink>.
+* Frozen and slow frame rendering - learn more in <PlatformLink to="/performance/instrumentation/automatic-instrumentation/#slow-and-frozen-frames">Slow and Frozen Frames</PlatformLink>.
```
Comment (lizokm) [inline src/platform-includes/performance/automatic-performance-metrics/apple.mdx:4]:
```suggestion
* <PlatformLink to="/performance/instrumentation/automatic-instrumentation/#app-start-tracking">Cold and warm app start time</PlatformLink>
* <PlatformLink to="/performance/instrumentation/automatic-instrumentation/#slow-and-frozen-frames">Frozen and slow frame rendering</PlatformLink>
```

File: src/platform-includes/performance/custom-performance-metrics/apple.mdx [hunk 2]
```diff
@@ -0,0 +1,32 @@
+Adding custom metrics is supported in Sentry's Cocoa SDK version `7.28.0` and above.
```
Comment (lizokm) [inline src/platform-includes/performance/custom-performance-metrics/apple.mdx:1]:
```suggestion
Adding custom metrics is supported in Sentry's Cocoa SDK, version `7.28.0` and above.
```

**Review by lizokm — APPROVED** [review]
Looks good, made a few formatting edits.

**Comment by imatwawana** [comment]
Hey Philipp, thanks for leaving this for us to review. @lizokm is our new docs manager!

**Pull request merged**